packages-next/types/src/config/index.ts

```diff
@@ -24,9 +24,12 @@ import type { ListAccessControl, FieldAccessControl } from './access-control';
 import type { ListHooks } from './hooks';
 
 export type KeystoneConfig = {
+  // e.g 'thinkmill:243809dsjkfdls-r2y8osdfjsdf-23y8osf',
+  cloud?: string;
```

Comment:
`apiKey`? or `apiSecret`? I like `apiKey` just because it's a commonly used thing that is instantly understandable to most people these days.